doc/manipulating_images/masker_objects.rst

```diff
@@ -176,14 +176,19 @@ Common data preparation steps: smoothing, filtering, resampling
 preparation::
 
    >>> from nilearn import input_data
-   >>> masker = input_data.NiftiMasker()
-   >>> masker
+   >>> masker = input_data.NiftiMasker() # doctest: +SKIP
+   >>> masker # doctest: +SKIP
    NiftiMasker(detrend=False, dtype=None, high_pass=None, low_pass=None,
          mask_args=None, mask_img=None, mask_strategy='background',
          memory=Memory(cachedir=None), memory_level=1, sample_mask=None,
          sessions=None, smoothing_fwhm=None, standardize=False, t_r=None,
          target_affine=None, target_shape=None, verbose=0)
```

Comment:
A better solution would be to use "# doctest: +ELLIPSIS" and replace "cachedir=None" by ellipsis: "...".

Reply:
I did not get the replacing cachedir. Could you please be even more details or reference to look at ?